src/include/cache.h

```diff
@@ -105,8 +108,13 @@ struct __wt_cache {
 	/*
 	 * Cache pool information.
 	 */
-	uint64_t cp_saved_read;		/* Read count from last pass */
-	uint64_t cp_current_read;	/* Read count from current pass */
+	uint64_t cp_saved_app_evicts;	/* User eviction count at last review */
+	uint64_t cp_saved_app_waits;	/* User wait count at last review */
+	uint64_t cp_saved_read;		/* Read count at last review */
+	uint64_t cp_pass_app_evicts;	/* User eviction count from this pass */
+	uint64_t cp_pass_app_waits;	/* User wait count from this pass */
+	uint64_t cp_pass_pressure;	/* Calculated pressure from this pass */
+	uint64_t cp_pass_read;		/* Read count from this pass */
```

Comment:
Consider reordering either for alphabetical or to group the "saved" with the "pass" for each category.